src/quo/components/list_items/network_list/style.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo.components.list-items.network-list.style
+  (:require [quo.foundations.colors :as colors]))
+
+(defn- background-color
+  [state customization-color theme]
+  (cond
+    (= state :pressed) (colors/resolve-color customization-color theme 5)
+    (= state :active)  (colors/resolve-color customization-color theme 10)
+    :else              :transparent))
```

Comment:
You can use `case` statement here. It is more efficient and cleaner when comparing a single variable against multiple values

```suggestion
  (case state
    :pressed (colors/resolve-color customization-color theme 5)
    :active  (colors/resolve-color customization-color theme 10)
    :transparent))
```